core/debugger/debugger_marshalls.cpp

```diff
@@ -36,8 +36,7 @@
 #define CHECK_END(arr, expected, what) ERR_FAIL_COND_V_MSG((uint32_t)arr.size() > (uint32_t)expected, false, String("Malformed ") + what + " message from script debugger, message too long. Expected size: " + itos(expected) + ", actual size: " + itos(arr.size()))
 
 Array DebuggerMarshalls::ScriptStackDump::serialize() {
-	Array arr;
-	arr.push_back(frames.size() * 3);
+	Array arr{ frames.size() * 3 };
```

Comment:
Do we want to use this syntax, or this one?
```suggestion
	Array arr = { frames.size() * 3 };
```
I believe so far we've used mostly the latter for initializer lists.

Reply:
Are there any functional differences between the two?

Reply:
https://stackoverflow.com/questions/63113374/c-vector-initialization-list-vs-assignment
There's a slight difference, not sure if it matters though?

Reply:
Reading online it seems most people prefer without the `=`, which is my preference too

Reply:
Changed all occurrences, hopefully I didn't miss anything.

Though seems like we use the direct initialization in quite a few places, mostly for non-array types ([including cases like this](https://github.com/godotengine/godot/blob/594d64ec244b6b99321e2096987d4d69de4c8845/platform/macos/export/export_plugin.cpp#L1247)).